renderer/components/Form/messages.js

```diff
@@ -24,4 +24,6 @@ export default defineMessages({
   transaction_speed_description_medium: 'less than 3 hours',
   transaction_speed_description_slow: 'up to 24 hours',
   transaction_speed_description_slowest: 'more than 24 hours',
+  zero_amount_request:
+    'This appears to be a zero amount payment request. Zap does not support zero amount payment requests due to security concerns associated with them.',
```

Comment:
I'd remove the note about security concerns. Please can you update so that it reads:

`Unsupported lightning payment request: payment requests must include a non-zero amount.`

Reply:
@mrfelton alright, done.